ground/gcs/src/plugins/config/configautotunewidget.cpp

```diff
@@ -407,23 +395,309 @@ void ConfigAutotuneWidget::stuffShareForm(AutotuneFinalPage *autotuneShareForm)
     }
 }
 
+// XXX appropriately encapsulate all this crap
+#define ATFLASH_MAGIC 0x656e755480008041 /* A...Tune */
```

Comment:
Files is really only for two things, IMO:

1. This--- a giant lump of samples in a special-purpose log file-- it's not really a filesystem, just a header + set of points.  It isn't really a clean design to do it over uavobjects (requires something like asking for a range of samples by uavo and them showing up in some other object, plus receiving the control information).
2. Shuttling logs back (so uh, uavtalk stream in uavtalk heh).

Maybe also for downloading partitions without being in the bootloader, and for tunneling stream-oriented stuff like RTK.